src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -14,9 +14,11 @@ function DatasetVisualizer(props: Props): JSX.Element {
   const { dataset } = props;
 
   const supportedVis = useMemo(() => getSupportedVis(dataset), [dataset]);
-  const [activeVis, setActiveVis] = useState<Vis | undefined>(
-    dataset && supportedVis[supportedVis.length - 1]
-  );
+  const [activeVis, setActiveVis] = useState<Vis>();
+
+  useEffect(() => {
+    setActiveVis(supportedVis[supportedVis.length - 1]);
```

Comment:
no out-of-bounds exception in JS, so `[][-1]` returns `undefined` which is fine by us